CMakeLists.txt

```diff
@@ -109,6 +109,8 @@ endif(Windows)
 # C++ Binding definition
 # ----------------------
 if(ENABLE_CXX)
+    set(CMAKE_CXX_STANDARD 11)
```

Comment:
It's better to set the property on the targets in bindings/cpp/CMakeLists.txt with set_target_properties instead of globally.
something like:
`set_target_properties(tuntap++ PROPERTIES CXX_STANDARD 11)`